### PR TITLE
misc updates

### PR DESCRIPTION
**Pull request by tarilabs**
- add GHA
- amend instructions

### CODE REVIEW

File: GET_STARTED.md [hunk 1]
```diff
@@ -62,6 +65,7 @@ spec:
   container:
     name: storage-initializer
     image: quay.io/$USER/oci-storage-initializer:${VERSION}
+    imagePullPolicy: IfNotPresent # NOT FOR PROD but allow easier testing of local images with KinD (just remove for prod)
```
Comment (lampajr) [inline GET_STARTED.md:68]: I remember I did the same for the model registry csi but this time I forgot. Thanks a lot!

**Review by lampajr — APPROVED** [review]
That's awesome, thanks a lot @tarilabs !

**Comment by lampajr** [comment]
@tarilabs FYI the GHA worked like a charm: https://github.com/lampajr/oci-storage-initializer/actions/runs/9402454029/job/25896621287

**Comment by tarilabs** [comment]
> @tarilabs FYI the GHA worked like a charm: https://github.com/lampajr/oci-storage-initializer/actions/runs/9402454029/job/25896621287

yeah, did some "local" testing first :) https://github.com/tarilabs/oci-storage-initializer/actions
😅

**Pull request merged**